src/libbson/tests/json/bson_corpus/array.json

```diff
@@ -14,16 +14,22 @@
             "canonical_extjson": "{\"a\" : [{\"$numberInt\": \"10\"}]}"
         },
         {
-            "description": "Single Element Array with index set incorrectly",
+            "description": "Single Element Array with index set incorrectly to empty string",
             "degenerate_bson": "130000000461000B00000010000A0000000000",
             "canonical_bson": "140000000461000C0000001030000A0000000000",
             "canonical_extjson": "{\"a\" : [{\"$numberInt\": \"10\"}]}"
         },
         {
-            "description": "Single Element Array with index set incorrectly",
+            "description": "Single Element Array with index set incorrectly to ab",
             "degenerate_bson": "150000000461000D000000106162000A0000000000",
             "canonical_bson": "140000000461000C0000001030000A0000000000",
             "canonical_extjson": "{\"a\" : [{\"$numberInt\": \"10\"}]}"
+        },
+        {
+            "description": "Multi Element Array with duplicate indexes",
```

Comment:
This test was introduced by https://github.com/mongodb/specifications/commit/863408ec4bec630b2b5b916e662aa4241f37fe90 for [DRIVERS-732](https://jira.mongodb.org/browse/DRIVERS-732). This should allow us to resolve [CDRIVER-3374](https://jira.mongodb.org/browse/CDRIVER-3374).